CHANGELOG.md

```diff
@@ -5,25 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+Contributors:
+
+- [@GitHub-Username](Link to GitHub profile) ([#PR-Number](Link to PR))
+- [@bolajiwahab](https://github.com/bolajiwahab) ([#1063])(https://github.com/sqlfluff/sqlfluff/pull/1063)
+- [@silverbullettruck2001](https://github.com/silverbullettruck2001) ([1126](https://github.com/sqlfluff/sqlfluff/pull/1126))
+- [@silverbullettruck2001](https://github.com/silverbullettruck2001) ([1099](https://github.com/sqlfluff/sqlfluff/pull/1099))
+
 ## [Unreleased]
+
+### Added
+
+- Support for the CREATE/DROP PROCEDURE statement for the mysql dialect [#901](https://github.com/sqlfluff/sqlfluff/issues/901))
+- Specific allowed/required syntaxes for CREATE/DROP FUNCTION within the mysql dialect [#901](https://github.com/sqlfluff/sqlfluff/issues/901))
+
 ### Changed
+
 - L034 now ignores select statements which contain macros.
 - L034 now ignores select statements part of a set expression, most commonly a union.
 - Respect XDG base dirs on Mac OS ([#889](https://github.com/sqlfluff/sqlfluff/issues/889)).
 - Fix bug [#1082](https://github.com/sqlfluff/sqlfluff/issues/1082), adding
   support for BigQuery `select as struct '1' as bb, 2 as aa` syntax
 
-Contributors:
-- [@GitHub-Username](Link to GitHub profile) ([#PR-Number](Link to PR))
-- [@bolajiwahab](https://github.com/bolajiwahab) ([#1063])(https://github.com/sqlfluff/sqlfluff/pull/1063)
-## [0.6.1] - 2021-06-01
 ### Changed
-- Added support for additional delimiters by creating a new DelimiterSegment in the 
```

Comment:
I see a couple of unrelated changes here, was it a merging issue?

Reply:
Yes, this was a merging issue